server/commands.go

```diff
@@ -19,7 +20,7 @@ import (
 )
 
 // add Rollapp commands
```

Comment:
wrong docstr

Reply:
It was a previous comment, but I'll add some comments to it